docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2058,6 +2058,40 @@ For example, consider a FHIR Server that is listening at https://fhir:9443/fhir-
 
 The originalRequestUriHeader is expected to contain the full path of the original request. Values with no scheme (e.g. `https://`) will be handled like relative URLs, but full URL values (including scheme, hostname, optional port, and path) are recommended. Query string values can be included in the header value but will be ignored by the server; the server will use the query string of the actual request to process the request.
 
+## 4.12.2 System Level History Usage and Concurrency Implications
```

Comment:
This is very good content that deserves to be somewhere.  Personally, I don't like the name of this document.  Its called a "user's guide" but I think its really more of an "administrator/operator's guide".

I think more of the end-user (FHIR Client-level) documentation exists in https://ibm.github.io/FHIR/Conformance and so I think I'd rather see this there.
Specifically, we need to update this line out there:
> history is only supported at the resource instance level (no resource type history and no whole-system history)

I think we can make these documentation-related edits post-merge.

Reply:
If needed, I think we can make these documentation-related edits post-merge.

Reply:
Perhaps we need a client developer guide. You can never have too many guides. Although as you say, perhaps the conformance guide fills that role.